internal/distributed/querynode/service.go

```diff
@@ -101,6 +101,9 @@ func (s *Server) init() error {
 
 	etcdCli, err := etcd.GetEtcdClient(
 		etcdConfig.UseEmbedEtcd.GetAsBool(),
+		etcdConfig.EtcdEnableAuth.GetAsBool(),
+		etcdConfig.EtcdAuthUserName.GetValue(),
```

Comment:
why do we need to change all test? 

by default we should not enable auth.
So GetEtcdClient do not need to add new params?

Reply:
this pr should change limited file as possible but current change involves too much unecessary changes.